homework_1/task_02/taxes/testCases_test.go

```diff
@@ -0,0 +1,46 @@
+package taxes_test
+
+type testCase struct {
+	input float64
+	output float64
+	error bool
```

Comment:
Attribute names here don't really help in understanding what they're supposed to be. They're too vague.